zipline/assets/assets.py

```diff
@@ -242,15 +345,20 @@ def populate_cache(self):
         # Wipe caches before repopulating
         self.cache = {}
         self.sym_cache = {}
+        self.future_cache = {}
         self.identifier_cache = {}
         self.fuzzy_match = {}
 
         counter = 0
         for identifier, row in self.metadata_cache.items():
-            self.spawn_asset(identifier=identifier, **row)
+            self._spawn_asset(identifier=identifier, **row)
             counter += 1
```

Comment:
Scott noticed that the counter is totally unused. Could you cut those variables as part of this PR?